Gemfile

```diff
@@ -15,3 +15,7 @@ if version == "master"
 else
   gem "rails", "~> #{version}.0"
 end
+
+group :test do
+  gem "timecop"
+end
```

Comment:
I don't think that would work because it wouldn't affect the `updated_at` timestamps that Rails adds on save.  You'd have to call `model.update_column(:updated_at, Time.new(2015,6,6))` after the object was created ( see https://github.com/rails/rails/blob/4-2-stable/activerecord/lib/active_record/persistence.rb#L271-L291 ) or whatever the id is in this case